src/Symfony/Component/Validator/Tests/Constraints/AbstractConstraintValidatorTest.php

```diff
@@ -244,7 +244,7 @@ protected function expectValidateValueAt($i, $propertyPath, $value, $constraints
 
     protected function assertNoViolation()
     {
-        $this->assertCount(0, $this->context->getViolations());
+        $this->assertSame(0, $violationsCount = count($this->context->getViolations()), sprintf('0 violation expected. Got %d.', $violationsCount));
```

Comment:
Ok, I understand what you mean. But in order to keep consistency between both "assertion", I think It's better to keep a "long" message:

```
1) SensioLabs\Bundle\MyBundle\Tests\Validator\Constraints\ProjectRepositoryUrlValidatorTest::testInternalRepositoryCanBeEmptyInitially
1 violation(s) expected. Got 0.
Failed asserting that 0 is identical to 1.

/home/greg/dev/github.com/sensiolabs/my_project/vendor/symfony/symfony/src/Symfony/Component/Validator/Tests/Constraints/AbstractConstraintValidatorTest.php:420
/home/greg/dev/github.com/sensiolabs/my_project/src/SensioLabs/Bundle/MyBundle/Tests/Validator/Constraints/ProjectRepositoryUrlValidatorTest.php:60
/home/greg/dev/github.com/sebastianbergmann/phpunit/src/TextUI/Command.php:151
/home/greg/dev/github.com/sebastianbergmann/phpunit/src/TextUI/Command.php:103

2) SensioLabs\Bundle\MyBundle\Tests\Validator\Constraints\ProjectRepositoryUrlValidatorTest::testInternalRepositoryCannotBeEmptyAfterFirstSync
0 violation expected. Got 1.
Failed asserting that 1 is identical to 0.

/home/greg/dev/github.com/sensiolabs/my_project/vendor/symfony/symfony/src/Symfony/Component/Validator/Tests/Constraints/AbstractConstraintValidatorTest.php:247
/home/greg/dev/github.com/sensiolabs/my_project/src/SensioLabs/Bundle/MyBundle/Tests/Validator/Constraints/ProjectRepositoryUrlValidatorTest.php:72
/home/greg/dev/github.com/sebastianbergmann/phpunit/src/TextUI/Command.php:151
/home/greg/dev/github.com/sebastianbergmann/phpunit/src/TextUI/Command.php:103
```